extensions/wikia/Search/WikiaSearch.setup.php

```diff
@@ -17,32 +16,36 @@
 /**
  * constants we want for search profiles
  */
-define( 'SEARCH_PROFILE_DEFAULT',  'default' );
-define( 'SEARCH_PROFILE_IMAGES',   'images' );
-define( 'SEARCH_PROFILE_USERS',    'users' );
-define( 'SEARCH_PROFILE_ALL',      'all' );
+define( 'SEARCH_PROFILE_DEFAULT', 'default' );
+define( 'SEARCH_PROFILE_IMAGES', 'images' );
+define( 'SEARCH_PROFILE_USERS', 'users' );
+define( 'SEARCH_PROFILE_ALL', 'all' );
 define( 'SEARCH_PROFILE_ADVANCED', 'advanced' );
 
 // autoloads values in the search namespace
-spl_autoload_register( function( $class ) {
-	if ( substr_count( $class, 'Wikia\\Search\\' ) > 0 ) {
-		$class = preg_replace( '/\\\\?Wikia\\\\Search\\\\/', '', $class );
-		$file = __DIR__ . '/classes/'.strtr( $class, '\\', '/' ).'.php';
-		if ( file_exists( $file ) ) {
-			require_once( $file );
-			return true;
+spl_autoload_register(
+	function ( $class ) {
+		if ( substr_count( $class, 'Wikia\\Search\\' ) > 0 ) {
+			$class = preg_replace( '/\\\\?Wikia\\\\Search\\\\/', '', $class );
+			$file = __DIR__ . '/classes/' . strtr( $class, '\\', '/' ) . '.php';
+			if ( file_exists( $file ) ) {
+				require_once( $file );
+
+				return true;
+			}
+
+			return false;
 		}
-		return false;
-	}  
-});
+	}
+);
 
 /**
  * Keeping the traditional controller registry for now
  */
-$wgAutoloadClasses['WikiaSearchController'] =  $dir . 'WikiaSearchController.class.php';
-$wgAutoloadClasses['WikiaSearchIndexerController'] =  $dir . 'WikiaSearchIndexerController.class.php';
-$wgAutoloadClasses['WikiaSearchAjaxController'] =  $dir . 'WikiaSearchAjaxController.class.php';
-$wgAutoloadClasses['WikiaSearchHelper'] =  $dir . 'WikiaSearchHelper.class.php';
+$wgAutoloadClasses['WikiaSearchController'] = $dir . 'WikiaSearchController.class.php';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiaSearchController" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)